ister_gui.py

```diff
@@ -3271,9 +3271,9 @@ def automatic_install(self):
         args = ister.handle_options(ister_cmd)
 
         ister.LOG = ister.logging.getLogger('ister')
-        ister.handle_logging(args.loglevel, ister_log, AlertLoggerHandler)
 
         try:
+            ister.handle_logging(args.loglevel, ister_log, AlertLoggerHandler(title))
```

Comment:
This change is unrelated to the rest of the commit. It should be in its own commit.

Reply:
ok

Reply:
ready